### ISSUE TITLE
Form answers reported as item in followup ticket notifications

### ISSUE THREAD

**Issue by kabassanov** [pluginsGLPI/formcreator#2451]
I have a customized notification template for ticket followups that includes ##ticket.numberofitems## and ##ticket.item.name## tags. Even if there is no visible items in the ticket web form, notifications display "Form answers - [Form Name]" and is counted in ##ticket.numberofitems##.

**To Reproduce**
1. Create a ntofication template for ticket followups including ##ticket.numberofitems## and ##ticket.item.name## tags
2. Fill a form that opens a ticket
3. Add a followup to the ticket
4. Check received notification

**Expected behavior**
Form answer should not be counted as item...

**Comment by btry** [comment]
Hi

When formcreator creates a ticket, this ticket is associated to an object representing the answers to a form. The relation also contains assets associates to this ticket. 

Identify the user and his profile when the ticket is created. In the profile, tab assistance you should find a list of types associatable to tickets. If form answer is in the list, remove it, then try to reproduce. Let me know if the count is correct.

**Comment by kabassanov** [comment]
Hi,

Form answer is not set to be associated to tickets in any profile.
I've just made a test and allowed it for the profile after the ticket creation. Now I can see the item in the items tab of the ticket in the web browser.

So it seems that association is (partially?) made even if it is not allowed, but remains hidden except for notifications. Well it is also logged in the ticket history tab.

**Comment by btry** [comment]
The relation cannot be made partially. The code in glpi which shows associated items filters them to show only the types added in the list I asked you to check.

By reading your result, I guess the code in the notification which counts the items does not filters them, while the list displayed by glpi does this filtering. 

I think that, for consistency, the filtering should be added to the notification.

**Comment by kabassanov** [comment]
I agree with this. Is this filtering part of the plugin or once the plugin is registered into GLPI, GLPI has to manage notification filtering?

**Comment by btry** [comment]
Hi

The filtering is in GLPI  and relies on the types associatable to tickets. I don't know yet if it is possible to apply some kind of filtering via Formcreator. This needs investigation.
